### DISCUSSION TITLE
WT32-SC01 can't get it to work

### DISCUSSION THREAD

**Discussion by dreamer1976** [arendst/Tasmota#17334] [General]
I'm trying to get a WT32-SC01 es32 with display working but I don't get it to work.
Flashing with esp32-tux works so the display is ok
What I've tried so far:

I have flashed the latest tasmota display.
And added the following code via configure other.
{"NAME":"WT32-SC01","GPIO":[6210,1,1,1,1,1,0,0,1,704,736,768,1,1,640,608,1,800,1024,992,0,1,1, 1,0,0,0,0,1,1,1,1,1,0,0,1],"FLAG":0,"BASE":1}
When I look at configure template everything seems to be fine.
GPIO23 is also set to backlight but I get no image at all.
Can someone tell me what I'm doing wrong?

tasmota works, so only the display with touch does not work

**Comment by sfromis** [comment]
Since you are configuring for [uDisplay](https://tasmota.github.io/docs/Displays/#universal-display-driver) with `Option A3`, you should also have a display descriptor file to match your display, probably the provided descriptor for [WT32_SC01](https://github.com/arendst/Tasmota/blob/development/tasmota/displaydesc/WT32_SC01.ini) as `display.ini` in the local Tasmota file system on the ESP32.

**Reply by dreamer1976** [reply]
thank you for the response
I thought that I only had to put in the template since this module is on the site.
https://templates.blakadder.com/wireless_tag_WT32-SC01.html

**Reply by sfromis** [reply]
Template has nothing to indicate what type of display it is.

**Reply by dreamer1976** [reply]
I can't find out exactly what the specs are of the display. When I google it I usually find the following
Device : WT32-SC01
Display / Touch Drivers : LovyanGFX
Widgets : LVGL 8.x

ESP-IDF => WT32-SC01 + LVGL8 + LOVYANGFX
[https://github.com/sukesh-a...](https://disq.us/url?url=https%3A%2F%2Fgithub.com%2Fsukesh-ak%2FLVGL8-WT32-SC01-IDF%3Ap-XrGa4j-d2aFccX9doCCpfj0Qs&cuid=6153168)

Arduino => WT32-SC01 + LVGL8 + LOVYANGFX
[https://github.com/sukesh-a...](https://disq.us/url?url=https%3A%2F%2Fgithub.com%2Fsukesh-ak%2FLVGL8-WT32-SC01-Arduino%3AQYW2kuKc4X45sqjzyHWXtPlM90k&cuid=6153168)

if I use sukesh's github and put his project in it then I have a picture and the touchscreen works.

If I use the Tasmota32 LVG bin then I can set DisplayModel to 14. if I then upload display.ini via manage file system, I have a brief image at startup

**Reply by dreamer1976** [reply]
Sorry i mean DisplayModel 17

**Reply by sfromis** [reply]
I already told you what to do, in addition to the template.
You should not need to mess around with setting specs, when you have those in that `display.ini` file.

When I posted "Template has nothing to indicate what type of display it is", that was meant as an explanation of why you also need the `display.ini` file, which I linked.

**Reply by sfromis** [reply]
When it is uDisplay, DisplayModel should always become 17, not anything else, and having `Option A3` in the template enables the uDisplay driver, as mentioned in the link I posted.

**Reply by sfromis** [reply]
The Berry example from https://tasmota.github.io/docs/LVGL/#using-lvgl is a nice test to see if it all works, to get more than the brief splash image at boot. Note that you may need to power on the display (Tasmota front page or power command). Of course, ignore the template and `display.ini` from the link, when you already have the right values.

**Reply by dreamer1976** [reply]
little bit stupid but "Note that you may need to power on the display" did the job.
Thank you i have now display en i see a reaction in berry scripting console when i push a button.
Now i have to find out how i can send a command like MQTT .
Thanks for the help
